<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="jest tests" tests="1" failures="1" errors="0" time="5.913">
  <testsuite name="Test the root path" errors="0" failures="1" skipped="0" timestamp="2024-01-14T08:48:35" time="5.842" tests="1">
    <testcase classname="Test the root path It should respond to the GET method" name="Test the root path It should respond to the GET method" time="0">
      <failure>Error: thrown: &quot;Exceeded timeout of 5000 ms for a hook.
Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test.&quot;
    at /home/ubuntu/Jenkinstask/__tests__/app.test.js:7:3
    at _dispatchDescribe (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/index.js:98:26)
    at describe (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/index.js:60:5)
    at Object.&lt;anonymous&gt; (/home/ubuntu/Jenkinstask/__tests__/app.test.js:4:1)
    at Runtime._execModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1646:24)
    at Runtime._loadModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1185:12)
    at Runtime.requireModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1009:12)
    at jestAdapter (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapter.js:79:13)
    at runTestInternal (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/runTest.js:389:16)
    at runTest (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/runTest.js:475:34)</failure>
    </testcase>
    <testcase classname=" Test execution failure: could be caused by test hooks like &apos;afterAll&apos;." name=" Test execution failure: could be caused by test hooks like &apos;afterAll&apos;." time="0">
      <failure>{&quot;message&quot;:&quot;&quot;,&quot;stack&quot;:&quot;Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (net.js:1310:16)\n    at listenInCluster (net.js:1358:12)\n    at Server.listen (net.js:1444:7)\n    at Function.listen (/home/ubuntu/Jenkinstask/node_modules/express/lib/application.js:635:24)\n    at Object.&lt;anonymous&gt; (/home/ubuntu/Jenkinstask/server.js:8:5)\n    at Runtime._execModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1646:24)\n    at Runtime._loadModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1185:12)\n    at Runtime.requireModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1009:12)\n    at Runtime.require [as requireModuleOrMock] (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1210:21)\n    at Object.&lt;anonymous&gt; (/home/ubuntu/Jenkinstask/__tests__/app.test.js:2:13)\n    at Runtime._execModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1646:24)\n    at Runtime._loadModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1185:12)\n    at Runtime.requireModule (/home/ubuntu/Jenkinstask/node_modules/jest-runtime/build/index.js:1009:12)\n    at jestAdapter (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapter.js:79:13)\n    at runTestInternal (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/runTest.js:389:16)\n    at runTest (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/runTest.js:475:34)\n    at TestRunner.runTests (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/index.js:101:12)\n    at TestScheduler.scheduleTests (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/TestScheduler.js:333:13)\n    at runJest (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/runJest.js:404:19)\n    at _run10000 (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/cli/index.js:320:7)\n    at runCLI (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/cli/index.js:173:3)\n    at Object.run (/home/ubuntu/Jenkinstask/node_modules/jest-cli/build/cli/index.js:155:37)\nError: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (net.js:1310:16)\n    at listenInCluster (net.js:1358:12)\n    at Server.listen (net.js:1444:7)\n    at Function.listen (/home/ubuntu/Jenkinstask/node_modules/express/lib/application.js:635:24)\n    at /home/ubuntu/Jenkinstask/__tests__/app.test.js:8:18\n    at Promise.then.completed (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/utils.js:381:26)\n    at new Promise (&lt;anonymous&gt;)\n    at callAsyncCircusFn (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/utils.js:316:10)\n    at _callCircusHook (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/run.js:181:40)\n    at _runTestsForDescribeBlock (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/run.js:47:7)\n    at _runTestsForDescribeBlock (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/run.js:60:9)\n    at run (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/run.js:25:3)\n    at runAndTransformResultsToJestFormat (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapterInit.js:170:21)\n    at jestAdapter (/home/ubuntu/Jenkinstask/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapter.js:82:19)\n    at runTestInternal (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/runTest.js:389:16)\n    at runTest (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/runTest.js:475:34)\n    at TestRunner.runTests (/home/ubuntu/Jenkinstask/node_modules/jest-runner/build/index.js:101:12)\n    at TestScheduler.scheduleTests (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/TestScheduler.js:333:13)\n    at runJest (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/runJest.js:404:19)\n    at _run10000 (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/cli/index.js:320:7)\n    at runCLI (/home/ubuntu/Jenkinstask/node_modules/@jest/core/build/cli/index.js:173:3)\n    at Object.run (/home/ubuntu/Jenkinstask/node_modules/jest-cli/build/cli/index.js:155:37)\nError [ERR_SERVER_NOT_RUNNING]: Server is not running.\n    at Server.close (net.js:1598:12)\n    at Object.onceWrapper (events.js:420:28)\n    at Server.emit (events.js:314:20)\n    at emitCloseNT (net.js:1651:8)\n    at processTicksAndRejections (internal/process/task_queues.js:83:21)&quot;}</failure>
    </testcase>
  </testsuite>
</testsuites>